Seminar_03/task_02.py

```diff
@@ -0,0 +1,7 @@
+# Напишите программу, которая найдёт произведение пар чисел списка.
+# Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+# Пример:
+# - [2, 3, 4, 5, 6] => [12, 15, 16];
+# - [2, 3, 5, 6] => [12, 15]
+lst = [2, 3, 4, 5, 6]
+print(lst, '->', [lst[i] * lst[-(i + 1)] for i in range(int(len(lst) / 2 + 0.5))])
```

Comment:
```suggestion
print(lst, '->', [lst[i] * lst[-i - 1)] for i in range((len(lst) + 1) // 2)])
```
На мой взгляд, так немного проще.